### PR TITLE
Add docs for per capp logging

### PR DESCRIPTION
**Pull request by chanikag**
This is related to https://github.com/wso2/micro-integrator/issues/3429

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/develop/monitoring-capp-level-logs.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Monitoring Composite Application-Level Logs
+
+The advantage of having per-composite application logs is that it is very easy to analyze/monitor what went wrong in this particular composite application (integration) by looking at the logs. 
+
+## Enabling log4j2 for capp logging
+
+Follow the instructions below to enable log4j2 logs for a sample composite application (named `HelloWorldCompositeExporter`).
+
+1.  Open up the `log4j2.properties` file (stored in the `<MI_HOME>/conf` directory.
```
Comment (anupama-pathirage) [inline en/docs/develop/monitoring-capp-level-logs.md:9]:
```suggestion
1. Open up the `log4j2.properties` file (stored in the `<MI_HOME>/conf` directory. 
```

File: en/docs/develop/monitoring-capp-level-logs.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Monitoring Composite Application-Level Logs
+
+The advantage of having per-composite application logs is that it is very easy to analyze/monitor what went wrong in this particular composite application (integration) by looking at the logs. 
+
+## Enabling log4j2 for capp logging
+
+Follow the instructions below to enable log4j2 logs for a sample composite application (named `HelloWorldCompositeExporter`).
+
+1.  Open up the `log4j2.properties` file (stored in the `<MI_HOME>/conf` directory. 
+2.  Let's modify the layout pattern of a new or existing appender for the `HelloWorldCompositeExporter` composite application by adding the following section.
```
Comment (anupama-pathirage) [inline en/docs/develop/monitoring-capp-level-logs.md:10]:
```suggestion
2. Let's modify the layout pattern of a new or existing appender for the `HelloWorldCompositeExporter` composite application by adding the following section.
```

File: en/docs/develop/monitoring-capp-level-logs.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    appender.CARBON_LOGFILE.strategy.max = 20
+    appender.CARBON_LOGFILE.filter.threshold.type = ThresholdFilter
+    appender.CARBON_LOGFILE.filter.threshold.level = DEBUG
+    ```
+
+3. Save the `log4j2.properties` file.
+
+A sample log output:
+
+    ```bash
```
Comment (anupama-pathirage) [inline en/docs/develop/monitoring-capp-level-logs.md:40]: It seems this is not properly getting rendered due to tab?

image

File: en/docs/develop/monitoring-capp-level-logs.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# Monitoring Composite Application-Level Logs
+
+The advantage of having per-composite application logs is that it is very easy to analyze/monitor what went wrong in this particular composite application (integration) by looking at the logs.
```
Comment (anupama-pathirage) [inline en/docs/develop/monitoring-capp-level-logs.md:3]:
```suggestion
The advantage of having per-composite application logs is that it is straightforward to analyze/monitor what went wrong in a particular composite application (integration) by looking at the logs. 
```

File: en/docs/develop/monitoring-capp-level-logs.md [hunk 5]
```diff
@@ -1,26 +1,28 @@
 # Monitoring Composite Application-Level Logs
 
-The advantage of having per-composite application logs is that it is very easy to analyze/monitor what went wrong in this particular composite application (integration) by looking at the logs. 
+The advantage of having per-composite application logs is that it is straightforward to analyze/monitor what went wrong in a particular composite application (integration) by looking at the logs. 
 
-## Enabling log4j2 for capp logging
+## Enabling log4j2 for Composite Application logging
```
Comment (anupama-pathirage) [inline en/docs/develop/monitoring-capp-level-logs.md:5]: Titles should be in sentence case. [1] https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-titles

File: en/docs/develop/monitoring-capp-level-logs.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    appender.CARBON_LOGFILE.strategy.max = 20
+    appender.CARBON_LOGFILE.filter.threshold.type = ThresholdFilter
+    appender.CARBON_LOGFILE.filter.threshold.level = DEBUG
+    ```
+
+3. Save the `log4j2.properties` file.
+
+## Sample log output
+
+Following is a sample log output
```
Comment (DinithiDiaz) [inline en/docs/develop/monitoring-capp-level-logs.md:42]:
```suggestion
Following is a sample log output:
```

File: en/docs/develop/monitoring-capp-level-logs.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
+# Monitor Composite Application-Level Logs
+
+The advantage of having per-composite application logs is that it is straightforward to analyze/monitor what went wrong in a particular composite application (integration) by looking at the logs. 
+
+## Enable log4j2 for composite application logging
+
+Follow the instructions below to enable log4j2 logs for a sample composite application (named `HelloWorldCompositeExporter`).
+
+1. Open up the `log4j2.properties` file (stored in the `<MI_HOME>/conf` directory.
```
Comment (DinithiDiaz) [inline en/docs/develop/monitoring-capp-level-logs.md:9]:
```suggestion
1. Open the `log4j2.properties` file located in the `<MI_HOME>/conf` directory. 
```

File: en/docs/develop/monitoring-capp-level-logs.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
+# Monitor Composite Application-Level Logs
+
+The advantage of having per-composite application logs is that it is straightforward to analyze/monitor what went wrong in a particular composite application (integration) by looking at the logs. 
+
+## Enable log4j2 for composite application logging
+
+Follow the instructions below to enable log4j2 logs for a sample composite application (named `HelloWorldCompositeExporter`).
+
+1. Open the `log4j2.properties` file located in the `<MI_HOME>/conf` directory. 
+2. Let's modify the layout pattern of a new or existing appender for the `HelloWorldCompositeExporter` composite application by adding the following section.
```
Comment (DinithiDiaz) [inline en/docs/develop/monitoring-capp-level-logs.md:10]:
```suggestion
2. Modify the layout pattern of a new or existing appender for the `HelloWorldCompositeExporter` composite application by adding the following section.
```

File: en/docs/develop/monitoring-capp-level-logs.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
+# Monitor Composite Application-Level Logs
+
+The advantage of having per-composite application logs is that it is straightforward to analyze/monitor what went wrong in a particular composite application (integration) by looking at the logs. 
+
+## Enable log4j2 for composite application logging
```
Comment (DinithiDiaz) [inline en/docs/develop/monitoring-capp-level-logs.md:5]:
```suggestion
## Enable Log4j2 for composite application logging
```

File: en/docs/develop/monitoring-capp-level-logs.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+Follow the instructions below to enable log4j2 logs for a sample composite application (named `HelloWorldCompositeExporter`).
```
Comment (DinithiDiaz) [inline en/docs/develop/monitoring-capp-level-logs.md:7]:
```suggestion
Follow the instructions below to enable Log4j2 logs for a sample composite application (named `HelloWorldCompositeExporter`).
```

**Review by DinithiDiaz — APPROVED** [review]

**Pull request merged**